### PR TITLE
Feature/tm1 memo 2

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**

### CODE REVIEW

**Comment by elizabeth-jimenez** [comment]
PR Artificially bloated cause branched off of Jeanie's TM1 PR. I would hold off on reviewing until that one merges to dev and then we update this branch with it - that way it'll be easier to see only what was changed on this PR

File: src/Components/Assignments/AssignmentCard/AssignmentCard.jsx [hunk 1]
```diff
@@ -1,7 +1,10 @@
+/* eslint-disable */
```
Comment (elizabeth-jimenez) [inline src/Components/Assignments/AssignmentCard/AssignmentCard.jsx:1]: 👀 rip

File: src/Components/Assignments/NotificationCard/Tabs/Routing.jsx [hunk 2]
```diff
@@ -0,0 +1,141 @@
+import Linkify from 'react-linkify';
+import TextareaAutosize from 'react-textarea-autosize';
+import { Row } from 'Components/Layout';
+
+const dummyArr = [{ code: 'd', name: 'SecState WashDC' }, { code: 'v', name: '11.1' }, { code: 'p', name: 'Publishable Positions' }, { code: 't', name: 'talentMAP' }];
```
Comment (elizabeth-jimenez) [inline src/Components/Assignments/NotificationCard/Tabs/Routing.jsx:5]: just UI, so im spending very little time on faking it out more

File: src/Components/Assignments/AssignmentCard/AssignmentCard.jsx (hunk 1 above)
Comment (elizabeth-jimenez) [reply src/Components/Assignments/AssignmentCard/AssignmentCard.jsx]: addressed

**Comment by elizabeth-jimenez** [comment]
Looks good to me!

**Comment by M00staff** [comment]
> Looks good to me!

I'll take your word for it - i have zero context for this one, and couldn't even find the page on the site 

... we gotta make sure to add that info in the PR description - can't expect everyone to have the same context

**Comment by elizabeth-jimenez** [comment]
> > Looks good to me!
> 
> I'll take your word for it - i have zero context for this one, and couldn't even find the page on the site
> 
> ... we gotta make sure to add that info in the PR description - can't expect everyone to have the same context

I agree, missed adding that context on this one bc i'm trying to juggle a lot of things, that's on me. However, we also have to be self-sustaining: a quick look at our jira board, filter by me, and it's readily apparent which ticket this PR is for. I do similar things all the time for fellow team-members - especially if I feel like they may be over-burdened atm.

**Pull request merged**